tests/Fixtures/app/config/config.yml

```diff
@@ -19,9 +20,18 @@ framework:
 
 doctrine:
     dbal:
-        driver:                        'pdo_sqlite'
-        path:                          '%kernel.cache_dir%/db.sqlite'
-        charset:                       'UTF8'
+        default_connection: '%default_connection%'
```

Comment:
I made an attempt with `%env(DEFAULT_CONNECTION)%` which would've been cleaner IMO but because this is an alias it doesn't work, see https://github.com/symfony/symfony/issues/21609

Reply:
Why don't you use different environments with multiple config files instead (`config_test.yml`, `config_test_mysql.yml`, `config_test_pgsql`)? Then you just have to set the `APP_ENV` environment variable in your Travis config instead of doing `cp`!

Reply:
would this mean that I'd have to copy the whole configuration file 3 times?

Reply:
Nop:
* `config_test.yaml`: the whole configuration + 
```yaml
doctrine:
      dbal:
         driver:                        'pdo_sqlite'
         path:                          '%kernel.cache_dir%/db.sqlite'
         charset:                       'UTF8'
```
* `config_test_mysql.yaml`:
```yaml
imports:
    - { resource: config_test.yaml }

doctrine:
    dbal:
        driver: pdo_mysql
        path: ~
        dbname:   'api_platform_test'
        user:     'travis'
        password: ''
        host:     'localhost'
```
etc.